dbt/include/global_project/macros/materializations/snapshots/helpers.sql

```diff
@@ -34,7 +34,12 @@
   {{ adapter.dispatch('snapshot_staging_table', 'dbt')(strategy, source_sql, target_relation) }}
 {% endmacro %}
 
+{% macro get_snapshot_table_column_names() %}
+    {{ return({'dbt_valid_to': 'dbt_valid_to', 'dbt_valid_from': 'dbt_valid_from', 'dbt_scd_id': 'dbt_scd_id', 'dbt_updated_at': 'dbt_updated_at'}) }}
+{% endmacro %}
+
 {% macro default__snapshot_staging_table(strategy, source_sql, target_relation) -%}
+    {% set stcn = config.get('snapshot_table_column_names') or get_snapshot_table_column_names() %}
 
```

Comment:
Instead of passing around `stcn`, which is tricky to grok it's semantic meaning as it's an acronym, would it be simpler to just assign them to variables immediately? 
```suggestion
{% set dbt_updated_at = stcn.dbt_updated_at %}
```

Reply:
Alternative we could just rename `stcn` to something like `columns`, I think it's reasonable to for the reader to infer these are snapshot columns from the context  of being in the `snapshot_staging_table` macro

Reply:
There is code that uses all 4 of the column names (and there's another one coming...) so it feels more succinct to use the dictionary. The places where there's only one used are more arguable, but I'd rather do it the same in every place.

I've renamed stcn to "column".